decidim-collaborative_texts/lib/decidim/collaborative_texts/component.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+Decidim.register_component(:collaborative_texts) do |component|
+  component.engine = Decidim::CollaborativeTexts::Engine
+  component.admin_engine = Decidim::CollaborativeTexts::AdminEngine
+  component.icon = "media/images/decidim_collaborative_texts.svg"
+  component.icon_key = "draft-line"
+  # component.permissions_class_name = "Decidim::CollaborativeTexts::Permissions"
+
+  # component.query_type = "Decidim::CollaborativeTexts::CollaborativeTextsType"
+
+  # component.register_stat ...
+
+  component.actions = %w(create update destroy)
+
+  component.settings(:global) do |settings|
+    settings.attribute :announcement, type: :text, translated: true, editor: true
+    settings.attribute :comments_blocked, type: :boolean, default: false
```

Comment:
I would like to keep the same structure as with other components
```suggestion
    settings.attribute :comments_enabled, type: :boolean, default: true
    settings.attribute :comments_max_length, type: :integer, required: true
```